package.json

```diff
@@ -68,6 +68,7 @@
     "request": "2.34.0",
     "request-promise": "4.2.1",
     "serve-favicon": "^2.5.0",
+    "semver": "7.3.5",
```

Comment:
I added this package which is very popular because it lets you things like this:

```
if semver.lt('1.13.6, config.client.version.version) // true
  do something if version is less than 1.13.6
else
  do something if version is more than 1.13.6
```

We probably don't want to lean to hard into these types of checks in case we do switch to semantic versioning.  I added this for the case of CustomerSpecificServerFunctions and accounting for backwards compatibility.